spec/models/order_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  it "is valid with valid attributes" do
+    expect(Order.new).to_not be_valid
+  end
+
+  describe "Scopes" do
+    let!(:user){FactoryBot.create :user, name: "tuongabc"}
+    let!(:order_1){FactoryBot.create :order,
+                                      user_name_at_order: user.name,
+                                      status: Settings.order_shipping,
+                                      user_id: user.id}
+    let!(:order_2){FactoryBot.create :order,
+                                     user_name_at_order: user.name,
+                                     status: Settings.order_delivered,
+                                     user_id: user.id}
+
+    context "with scope newest" do
```

Comment:
phân biệt describer context và it

Reply:
describe dùng để nhóm các test case theo class, module, method hoặc theo action của controller.
+ Cần có mô tả cho trường hợp không là method.
+ Trong trường hợp đối với instance method thì gắn thêm "#" ví dụ "#method".
+ Trong trường hợp đối với class method thì gắn thêm "." ví dụ ".method"

context dùng để nhóm các điều kiện của example.
+ Phần mô tả của context thường bắt đầu bằng when hoăc with 

it được sử dụng để định nghĩa một "Ví dụ" hay nói cách khác là dùng để định nghĩa 1 test case.

Reply:
Chỗ này em nên đổi context thành describer vì Scope như là một class method